packages/storybook/stories/wc-helpers.jsx

```diff
@@ -129,6 +130,21 @@ export function Guidance({ href, name }) {
   );
 }
 
+const getDocsTagValue = (tagName, docsTags = []) => {
+  return docsTags
+    .filter(item => item.name === tagName)
+    .map(item => item.text)
+}
```

Comment:
Can we also make use of this function here:

https://github.com/department-of-veterans-affairs/component-library/blob/ccc6d2d88c567db08e6e17d512230dea8a76f7f4/packages/storybook/stories/wc-helpers.jsx#L196-L198